### ISSUE TITLE
Spinner should have message

### ISSUE THREAD

**Issue by taysea** [grommet/eslint-plugin-grommet#15]
**Please describe what the rule should do:**

Spinner should have message that is announced by screenreader

**What category of rule is this? (place an "X" next to just one item)**

[x] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```js
<Spinner />
```

**Why should this rule be included in eslint-plugin-grommet?**

Accessibility best practice.
